docs/guides/cloud/data-storage-and-masking.mdx

```diff
@@ -0,0 +1,168 @@
+---
+title: Data Storage and Controls
+sidebar_position: 60
+---
+
+:::info
+
+## <Icon name="graduation-cap" /> What you'll learn
+
+- What data Cypress Cloud stores
+- What controls exist to control data sent to and viewed in Cypress Cloud
+
+:::
+
+## Data Storage
+
+:::caution
+
+Please refer to the [Cypress Cloud Terms of Use](https://cloud.cypress.io/terms-of-use) and our [Security & Compliance](https://www.cypress.io/security/) guide for more specifics around reasonable use of Cypress Cloud, data storage and security at Cypress.
+
+:::
+
+When you run Cypress via `cypress run` and pass the `--record` flag, the following data is stored in Cypress Cloud for every run:
+
+- Standard output in the terminal
+- Test results
+- Test definitions
+- Cypress configuration (minus Cypress environment variables)
+- Screenshots
+- Videos
+- OS environment variables related to CI and git information
+
+When [Test Replay](#Test-Replay) capturing is enabled, the additional data outlined below will be sent to the Cypress Cloud. You can disable capturing: TODO link this data.
+
+- The rendered DOM and CSS styles for the application under test
+- Cypress commands and events represented in the Command Log
+- Network traffic within your application under test
+- Browser console logs
+
+Cypress Cloud does not capture anything related to the code of your application under test or the code within any repositories that are associated with your project.
+
+## Cypress Cloud data controls
+
+We understand that controlling what data is sent to Cypress Cloud is important to our users. Cypress provides a number of ways to control what data is sent to Cypress Cloud.
+
+If you'd like more granular control over what data is sent to Cypress Cloud, please comment in this discussion: TODO:ADD A LINK TO DISCUSSION or [contact us](https://www.cypress.io/contact/) to discuss your needs.
+
+### Project access controls
+
+Cypress Cloud offers controls around the visibility of a project. You can set a project to be public or private. [See project access control settings.](/guides/cloud/account-management/projects#Public-vs-Private)
+
+Setting a project to private means that users invited to the organization will have visibility into the project, its runs, and any data captured by Cypress.
```

Comment:
```suggestion
Setting a project to private means that only the users explicitly invited to the organization will have visibility into the project, its runs, and any data captured by Cypress.
```
Contemplating whether the extra emphasis here helps to drive home the point. Take it if you like it.

Reply:
Addressed in [`eb566cc` (#5425)](https://github.com/cypress-io/cypress-documentation/pull/5425/commits/eb566cca4cf3b6e79c8aba1084cb8be1a495c69f)